docs/netdata-cloud/netdata-cloud-on-prem/installation.md

```diff
@@ -1,149 +1,121 @@
-| Default **storage class configured and working** | Persistent volumes based on SSDs are preferred                                                                                                                                                      |
-
-The following 3rd party components are used, which can be pulled with the `netdata-cloud-dependency` package we provide:
-
-| component              | description                                                       |
-|:-----------------------|:------------------------------------------------------------------|
-| **Ingress controller** | Supporting HTTPS                                                  |
-| **PostgreSQL**         | v13.7, the main database for all metadata the OPC maintains       |
-| **EMQX**               | v5.11, MQTT Broker that allows Agents to send messages to the OPC |
-| **Apache Pulsar**      | v2.10+, message broker for inter-container communication          |
-| **Traefik**            | v2.7.x, the internal API Gateway                                  |
-| **Elasticsearch**      | v8.8.x, stores the events feed                                    |
-| **Redis**              | v6.2, used for caching                                            |
-| **imagePullSecret**    | our ECR repos are secured                                         |
-
-> **Note**
->
-> Keep in mind that the pulled versions are not configured properly for production use.
->
-> Customers of OPC are expected to configure these applications according to their needs and policies for production use.
-
-**The following components are required for installation:**
-
-- AWS CLI
-- Helm version 3.12+ with OCI Configuration (explained in the installation section)
-- Kubectl
+## System Requirements
 
-**Minimum requirements:**
+**Core Infrastructure**:
 
-- 4 CPU cores
-- 15GiB of memory
-- Cloud services are ephemeral
+| Requirement           | Details                                                                                                                                        |
+|:----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------|
+| **Kubernetes**        | - Version 1.23 or newer<br/>- Metrics server installed (for autoscaling)<br/>- Default storage class configured (SSD-based preferred)          |
+| **TLS certificate**   | - Single certificate for all endpoints, or separate certificates for frontend, API, and MQTT<br/>- Must be trusted by all connecting entities. |
+| **Minimum Resources** | - 4 CPU cores<br/>- 15GiB memory<br/>- Note: Cloud services are ephemeral                                                                      |
 
-**The requirements for the non-production Dependencies helm chart:**
```

Comment:
@Ancairon I'm unclear about the "non-production Dependencies helm chart" requirements (what is that?). I'll ask @M4itee and update.